chart/prometheus-msteams/templates/configMapConfig.yaml

```diff
@@ -17,4 +17,9 @@ data:
 {{- if hasKey $connectorWithCustomTemplate "escape_underscores" }}
         escape_underscores: {{ $connectorWithCustomTemplate.escape_underscores }}
 {{- end }}
+{{- if hasKey $connectorWithCustomTemplate "retry_max" }}
+        retry_max: {{ $connectorWithCustomTemplate.retry_max }}
+{{- else }}
+        retry_max: 3
+{{- end}}
```

Comment:
We wouldn't need this if we use `retry-max` as a flag parameter (see comment below).

Reply:
I have halted my review on this line as this will change a lot of what you've changed afterwards.